doc/cephfs/client-config-ref.rst

```diff
@@ -176,9 +176,17 @@
 :Type: Boolean
 :Default: ``false``
 
+``fuse_default_permissions``
+
+:Description: When set to ``true``, the FUSE client enforces permissions. When set to ``false``, the ``ceph-fuse`` utility enforces the permissions. Set to `false` together with the ``client acl type=posix_acl`` option to enable POSIX ACL.
```

Comment:
Maybe it's this is clearer if we say something like "When set to ``false``, ceph-fuse does its own permissions checking, instead of relying on the permissions enforcement in FUSE.  Set to `false` together with the ``client acl type=posix_acl`` option to enable POSIX ACL."

Stylistically it's always nicer to define things in terms of their 'true' meaning, but in this instance all the important effects are what happens when its set to false.